Client/src/Unified/Services/Submitter/BatchUntilInactiveBlock.cs

```diff
@@ -0,0 +1,249 @@
+  /// </param>
+  /// <param name="executionDataFlowBlockOptions">Parameters to control execution for each block in pipeline</param>
+  public BatchUntilInactiveBlock(int                                       bufferRequestsSize,
+                                 TimeSpan                                  timeout,
+                                 [CanBeNull] ExecutionDataflowBlockOptions executionDataFlowBlockOptions = null)
+  {
+    executionDataFlowBlockOptions_ = executionDataFlowBlockOptions ?? new ExecutionDataflowBlockOptions
+                                                                      {
+                                                                        BoundedCapacity        = 1,
+                                                                        MaxDegreeOfParallelism = 1,
+                                                                        EnsureOrdered          = true,
+                                                                      };
+
+    source_ = new BatchBlock<T>(bufferRequestsSize,
+                                new GroupingDataflowBlockOptions
+                                {
+                                  BoundedCapacity = bufferRequestsSize,
+                                  EnsureOrdered   = true,
+                                });
+
+    timer_ = new Timer(_ =>
+                       {
+                         source_.TriggerBatch();
+                       },
+                       null,
+                       timeout,
+                       System.Threading.Timeout.InfiniteTimeSpan);
+
+    timeoutTransformBlock_ = new TransformBlock<T[], T[]>(value =>
+                                                          {
+                                                            timer_.Change(timeout,
+                                                                          System.Threading.Timeout.InfiniteTimeSpan);
+
+                                                            return value;
+                                                          },
+                                                          executionDataFlowBlockOptions_);
+
+    source_.LinkTo(timeoutTransformBlock_);
+
+    Timeout = timeout;
+  }
+
+  /// <summary>
+  ///   Simple Getter to return size of batch in the pipeline
+  /// </summary>
+  public int BatchSize
+    => source_.BatchSize;
+
+  /// <summary>
+  ///   Return the TimeSpan timer set in the constructor
+  /// </summary>
+  public TimeSpan Timeout { get; }
+
+  /// <summary>
+  ///   Task to check the completion
+  /// </summary>
+  public Task Completion
+    => source_.Completion;
+
+  /// <summary>
+  ///   Signal that the pipeline should be closed
+  /// </summary>
+  public void Complete()
+    => source_.Complete();
+
+  /// <summary>
+  ///   Return the exception produce in a Transform block or ActionBlock
+  /// </summary>
+  /// <param name="exception"></param>
```

Comment:
You need a description.
Maybe you can use directly the following:
```c#
/// <inherit_doc/>
```

Reply:
This is indeed better than copy pasting the doc